docker-cloud/apps/stack-yaml-reference.md

```diff
@@ -47,7 +47,7 @@ The image used to deploy this service. This is the only mandatory key.
 ```yml
 image: drupal
 image: dockercloud/hello-world
-image: quay.io/borja/redis
+image: my.registry.com/redis
```

Comment:
The biggest change here is that quay.io/borja/redis works and my.registry.com/redis doesn't.